src/Tizen.NUI/src/public/XamlBinding/XamlStyle.cs

```diff
@@ -0,0 +1,212 @@
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+using Tizen.NUI.StyleSheets;
+using System.ComponentModel;
+
+namespace Tizen.NUI.Binding
+{
+    /// <since_tizen> 6 </since_tizen>
```

Comment:
please remove <since_tizen> for hidden api.
also please remove the comment below for hidden api.

**/// This will be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API.**